src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  // eslint-disable-next-line no-param-reassign
+  const isMarried = person.partnerName;
```

Comment:
The `isMarried` variable is determined by checking `person.partnerName`. This might not be accurate if `partnerName` is an empty string or undefined. Consider using `person.isMarried` directly to determine if the person is married.